src/res_printer.ml

```diff
@@ -462,16 +462,23 @@ let printPolyVarIdent txt =
     | NormalIdent -> Doc.text txt
 
 
-let printLident l = match l with
+let printLident l =
```

Comment:
how about implementing a safe version of `Longident.flatten` here instead, which returns an option type, or even an empty list

this is what it does in the compiler libs:
```
let rec flat accu = function
    Lident s -> s :: accu
  | Ldot(lid, s) -> flat (s :: accu) lid
  | Lapply(_, _) -> Misc.fatal_error "Longident.flat"

let flatten lid = flat [] lid
```

Reply:
@amiralies happy to merge after that change.

Reply:
I'll check it later today